lib/datetime_helper/serialisers/active_model_serialiser_helper.rb

```diff
@@ -11,7 +11,7 @@ def enforce_zulu_time(attribute)
     # Enforces that a Date is serialized as iso8601 format
     def enforce_iso8601_date(attribute)
       define_method(attribute) do
-        object.send(attribute).iso8601 unless object.send(attribute).nil?
+        object.send(attribute).strftime('%F') unless object.send(attribute).nil?
```

Comment:
Ah okay I see. In which case update the comment to say it enforces that a Date or DateTime is serialised as an iso8601 formatted Date string.